fxa-oauth-server/test/grant.js

```diff
@@ -147,4 +146,95 @@ describe('validateRequestedGrant', () => {
       await validateRequestedGrant({ ...CLAIMS, 'fxa-tokenVerified': false }, CLIENT, requestedGrant);
     }, AppError, 'Requested scopes are not allowed');
   });
+
+});
+
+describe('generateTokens', () => {
+  let mockAccessToken;
+  let mockAmplitude;
+  let mockDB;
+
+  let generateTokens;
+  let requestedGrant;
+  let scope;
+
+  beforeEach(() => {
+    scope = ScopeSet.fromArray(['profile:uid', 'profile:email']);
+
+    mockAccessToken = {
+      expiresAt: Date.now() + 1000,
+      scope,
+      token: 'token',
+      type: 'access_token'
+    };
+
+    requestedGrant = {
+      clientId: 'foo',
+      scope,
+      userId: 'bar',
+    };
+
+    mockAmplitude = sinon.spy();
+    mockDB = {
+      generateAccessToken: sinon.spy(async () => mockAccessToken),
+      generateIdToken: sinon.spy(async () => ({ token: 'id_token' })),
+      generateRefreshToken: sinon.spy(async () => ({ token: 'refresh_token' } ))
+    };
+
+    generateTokens = proxyquire('../lib/grant', {
+      './metrics/amplitude': () => mockAmplitude,
+      './db': mockDB
+    }).generateTokens;
+  });
+
+  it('should required params in result', async () => {
```

Comment:
Is there a word missing between `should` and `required` here?

Reply:
Good spot, updated.